### PR TITLE
[Final submission] Essay: Agile Project Management with Jira

### PR DESCRIPTION
**Pull request by fbjorkman**
Looking forward to your feedback! @kittytinythai @elmaxe

### CODE REVIEW

**Comment by kittytinythai** [comment]
Thanks for the notification 😊 Me and @elmaxe will give you separate feedback. We'll submit it before the end of the day!

**Comment by elmaxe** [comment]
My feedback

# Feedback on essay "Agile project management with Jira"

Essay written by: Fredrik Björkman, Adam Henriksson
Feedback provided by: [Axel Elmarsson](https://github.com/elmaxe) (<axelelm@kth.se>)

## Overall

Good grammar, good spelling and good structure of the essay. For the front page I would consider capitalizing every word of the essay, since we are writing an essay in English ("Agile Project Management with Jira" instead of "Agile project management with Jira"). A minor detail: this probably differs from person to person and what kind of essay it is, but front pages don't always have a page numbers, do they?

Figures should be referred to [[1]](https://guides.lib.monash.edu/citing-referencing/apa-tables-figures), otherwise there is no meaning in having them. You are currently only referring to figure 1 and 2. The rest (3, 4, 5, 6) you need to refer to or remove completely.

Nice conclusion in the end! Didn't look too much at your sources, but they look credible, nice that you had many.

I will now give feedback session by session.

## 1 Introduction

"We will not go into how to set up..." -> "we will not go into detail about how to set up..." is better phrasing I think.

Other than that I have nothing to point out.

## 2 Background

Perhaps you should refer to Figure 1 earlier in the text. Currently you are referring to it after you have talked about agile methods (end of page 3). I think it would be better to refer to figure 1 in the end of the first paragraph, and only refer to figure 2 in the end of the third paragraph.

Nice figures, good figure texts, as mentioned earlier, good that you refer to these figures.

Content is good.

## 3.1 What is Jira

Perhaps you should add: "That question we will answer in the following question" or something in those lines after the last question in this section.

## 3.2 Organize tasks, introducing Kanban

Nothing to say about this section, looks good!

## 3.3 Roadmaps

Change from "and the amount of sprints you have to achieve that goal" to "and the amount of sprints you NEED TO have to achieve...", alternatively "the amount of sprints you have to achieve that goal in", I think something is missing in that sentence.

## 3.4 Knowledge management with Confluence

I think you shouldn't write "our product" and "we need to follow". I would prefer "What are the formal requirements of the product and the regulations that needs to be followed". The former is more personal, like you're talking about a product you have, but these are instructions for any project. In the following points for example, you are doing it correctly: You are not writing "what did our team do well..." for example.

## 4 Conclusion

I think it is a good conclusion, it sums up what you have talked about in the essay in a nice way.

# Summary

I haven't heard about Jira before so it was nice to get to see a nice agile work tool you can use. The essay's content is good, the structure follows the typical essay format. Great job (:

If you have any questions, feel free to ask!

**Comment by kittytinythai** [comment]
# Feedback on essay *Agile project management with Jira*
**Original authors:** Fredrik Björkman (fbjorkma), Adam Henriksson (adahen)

**Feedback submitter:** Kitty Thai (kittyt)

## General Feedback
The introduction contains a short and concise description of the main purpose of the essay. Even though it is short, it is still clear.

The essay addresses an interesting topic which also is relevant for DevOps. It may be obvious why the topic is related to DevOps, but it would still be beneficial to add a simple and concise sentence that explains this. Mostly because it will be more clear for the reader's understanding.

The background gives the reader an understandable view of going from traditional waterfall methodology to more agile approaches, as well as the need of tools such as Jira to improve and streamline the activities. The illustrations are very helpful and supports the written text very well 👍

The structure of the essay is a good foundation to continue working with. For instance the sections 3.2, 3.2.1, 3.2.2 address different methods used when planning work, and not so much of Jira itself. Therefore, a suggestion for improvement would be to put 3.2, 3.2.1 and 3.2.2 into a separate section before introducing the section about Jira.
Otherwise, good job and good luck with the last modifications before final submission 👍 

## Suggestions for Improvement
All the more detailed suggestions for improvement are listed below. Bold text corresponds to added words and crossed over words should be removed.

### 1 Introduction
Here are a few suggestions for change:

**Suggestion #1**
> In this essay, the agile project management tool Jira will be discussed ~and the~ **, as well as** advantages and disadvantages of using Jira in the software development process.

**Suggestion #2**
> We will not ~go into~ **discuss** how to set up and use specific features of Jira, but instead **focus on** the features of Jira, the problems Jira ~are~ **is** trying to solve, and the benefits of agile project management in workplaces that have embraced agile development practices.

### 2 Background

**Suggestion #3**
> This ~pretty much~ assumes that the developing team has complete and perfect information about the ~project's goal~ **goals of the project** and expected solution**s**.

**Suggestion #4**
> In modern software development, ~where~ radical changes in the customer requirements may occur frequently. **N**ew innovative ideas may arise from competitors or other unexpected events may happen that need to be addressed, **there is** a need for **flexibility** and the ability to react quickly is essential for the development process.

**Suggestion #5**
> With the need for a more agile development process, agile methodologies ~like~ **such as** Scrum and Kanban became more popular due to the more iterative approach of the software development [6].

### 3 The project management tool Jira

#### 3.1 What is Jira?
I have a question regarding the following sentence

> Then, we scale back and talk about the features that help managers plan and organize important milestones, get actionable data, achieve long-term goals, and win.

What are you meaning with "win"?

I think the second paragraph in this section belongs to the text under section 3. It is because this paragraph describes what the entire section 3 will address instead of what Jira is. Also the last sentence of this section would maybe suit the coming section 3.2, mostly because it is a good starting sentence for that section.

#### 3.2 Organize tasks, introducing Kanban

**Suggestion #6**
> ~If you cannot see what problems exist, it's hard to fix it.~ It's hard to fix problems if you cannot see them.

**Suggestion #7**
> Although, this system breaks down if there are too many planned tasks ~already planned~
that need to be completed, or even worse, too many tasks that never get completed.

#### 3.2.1 Limiting Work-In-Progress
This section contains a relevant discussion of why limiting the number of tasks that are work in process.

#### 3.2.2 Iterative development and sprints

**Suggestion #8**
> By using a ~Sccrum~ **Scrum** board template for your project instead of Kanban, you can group issues in "sprints".

**Suggestion #9**: team's instead of teams
> There are plenty of tools that managers can use to plan sprints and track the team's progress

### 3.3 Roadmaps

**Suggestion #10**
> Executives need a roadmap that ~places emphasis~ **emphasizes** on how development goals support high-level company goals.

**Suggestion #11**: I'm not sure if you mean "customer conversion" as when the staff needs to communicate with a customer as a stakeholder or customer support. Maybe this needs to be rewritten to prevent confusion for the reader.
> Sales personnel need a roadmap to support sales conversations about new
potential features and to assist in customer ~conversion~ support.

### 3.4 Knowledge management with Confluence

**Suggestion #12**: Maybe rephrase this sentence, I'm not sure what you are trying to say here. Especially the last part of the sentence.
> Con uence is designed to store knowledge from all parts of the organization
and **restrict access wherever necessary**.

### 4 Conclusion

**Suggestion #13**
> This suite of products deliver a powerful toolset for managers striving to implement **the** agile **workflow**, so teams can quickly learn, change and grow over time, while still meeting its obligations and long-term goals.

**Comment by fbjorkman** [comment]
Final submission made! We have acted on the feedback received and included the change log in the README. Thank you @kittytinythai and @elmaxe for good and constructive feedback!

**Comment by SophieHYe** [comment]
Thanks for the final essay submission. Could you please solve the conflicting files then we can merge your PR.

**Comment by fbjorkman** [comment]
@SophieHYe The conflict is now solved!

**Comment by SophieHYe** [comment]
Thanks for the update. I am merging your PR now.
